src/modules/localization/src/wheel_odom/odometry.cpp

```diff
@@ -132,16 +137,36 @@ namespace wheel_odometry_node
         yaw_estimate = previous_yaw + (vel_magnitude/WHEELBASE)*tan(steering_angle)*DT;
         previous_yaw = yaw_estimate;
         
-
         // velocity estimate in x and y
         x_velocity = vel_magnitude * cos(yaw_estimate);
         y_velocity = vel_magnitude * sin(yaw_estimate);
         
+        /*
+        // position estimate in x and y
+        x_position = previous_x_position + x_velocity*DT;
+        y_position = previous_y_position + y_velocity*DT;
+
+        // getting the previous x and y positions
+        previous_x_position = x_position;
+        previous_y_position = y_position;
+
+        // position messages
+        position_estimate.pose.pose.position.x = x_position;
+        position_estimate.pose.pose.position.y = y_position;
+
+        // position covariances
+        float x_pose_covariance = 0;
+        float y_pose_covariance = 0;
+
+        position_estimate.covariance[0] = x_pose_covariance;
+        position_estimate.covariance[7] = y_pose_covariance;
+        */
+
         // twist messages
         velocity_estimate.twist.linear.x = x_velocity;
         velocity_estimate.twist.linear.y = y_velocity;
-        //velocity_estimate.linear.z = 0;
-        // covariances 
+        
+        // velocity covariances 
         float x_vel_covariance = 0.0021878/2;

```

Comment:
How are these determined?

Reply:
Is there a particular reason why this is constant instead of calculated in (psuedo) real time?